bin/main.rb

```diff
@@ -1,85 +1,102 @@
-#!/usr/bin/env ruby 
+#!/usr/bin/env ruby
 
-class Player
-  attr_accessor :id
-  attr_accessor :name
-
-  def initialize(id, name)
-    @id = id
-    @name = name
-  end
-end
-
-class TicTacToe
-  attr_accessor :board
-  attr_accessor :players
-  attr_accessor :turn
-
-  def initialize(players)
-    @players = []
-    @players[0] = players[0]
-    @players[1] = players[1]
-    @board = (1..9).to_a
-    @turn = 0
-  end
-  
-  def set(num)
-    player_id = @players[@turn % 2].id
-    @board[num - 1] = player_id == 0 ? 'O' : 'X'
-  end
-
-  # check if the game is over
-  def game_over?
-    ['012', '345', '678', '036', '147', '258', '048', '246'].each do |l|
-      l_0, l_1, l_2 = l[0].to_i, l[1].to_i, l[2].to_i 
-        return true if @board[l_0] == @board[l_1] && @board[l_1] == @board[l_2]
-      
-    end
-    false
-  end
-
-end
+require './lib/tic_tac_toe'
+require './lib/player'
```

Comment:
[OPTIONAL]
Should use `require_relative` and not `require`:
Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory.